README.md

```diff
@@ -328,6 +328,92 @@ pipeline {
 }
 ```
 
+Here's how you install it using Pipelines:
+
+1. Make sure you have the connection details of your JFrog environment.
+2. Save the JFrog connection details as JFrog Platform Access Token integration named **jfrogPlatform**.
+3. Save your Bitbucket access token as a generic integration named **bitbucket** with the **token** as key to the Bitbucket access token.
```

Comment:
"... with the **token** as key to the Bitbucket access token." - this part isn'r clear.